practice-one/exercise-1/js/script.js

```diff
@@ -0,0 +1,27 @@
+// Change the body style so it has a font-family of "Arial, sans-serif".
+document.body.style.fontFamily = 'Arial, sans-serif';
+
+// Replace each of the spans (nickname, favorites, hometown) with your own information
+const nickName = document.querySelector('#nickname');
+nickName.textContent = 'Nguyen Van Hoa';
+const interests = document.querySelector('#favorites');
+interests.textContent = 'e-sport';
+const homeTown = document.querySelector('#hometown');
+homeTown.textContent = 'Quang Nam';
+
+// Iterate through each li and change the class to "listitem". 
+// Add a style tag that sets a rule for "listitem" to make the color red.
+const getElement = document.querySelectorAll('li');
```

Comment:
Which li used for? Show user information, right? So don't use getElement for it. The same for changeColor